interface/super/edit_layout_props.php

```diff
@@ -103,8 +107,8 @@ function submitProps() {
    <select name='form_columns'>
 <?php
   for ($cols = 2; $cols <= 10; ++$cols) {
-    echo "<option value='$cols'";
-    if ($cols == 4) echo " selected";
+    echo "<option value=";
+    echo (($cols == 4) ? "'' selected" : "'$cols'");
```

Comment:
Since you're changing this loop, would be nice if you could refactor so it's not unfriendly/unreadable. Maybe set $cols on a separate line and then echo the `<option>` line without any logic in it